src/main/java/pomPages/ProductPage.java

```diff
@@ -0,0 +1,12 @@
+package pomPages;
+
+public class ProductPage {
+	
+	/*
+	 * element to clickLink

```

Comment:
this page is no available in  this application

Reply:
i fixed the review comment